ydb/core/statistics/service/service_impl.cpp

```diff
@@ -969,31 +894,104 @@ class TStatService : public TActorBootstrapped<TStatService> {
         }
     }
 
+    void SendStatisticsRequest(const TActorId& clientId) {
+        auto request = std::make_unique<TEvStatistics::TEvStatisticsRequest>();
+        auto& record = request->Record;
+        record.MutableTypes()->Add(NKikimr::NStat::COUNT_MIN_SKETCH);
+
+        auto* path = record.MutableTable()->MutablePathId();
+        path->SetOwnerId(AggregationStatistics.PathId.OwnerId);
+        path->SetLocalId(AggregationStatistics.PathId.LocalPathId);
+
+        auto* columnTags = record.MutableTable()->MutableColumnTags();
+        for (const auto& tag : AggregationStatistics.ColumnTags) {
+            columnTags->Add(tag);
+        }
+
+        NTabletPipe::SendData(SelfId(), clientId, request.release(), AggregationStatistics.Round);
+    }
+
+    void OnTabletError(ui64 tabletId) {
+        LOG_DEBUG_S(TlsActivationContext->AsActorContext(), NKikimrServices::STATISTICS,
+            "Tablet " << tabletId << " is not local.");
```

Comment:
Is this message right when it is called from 958 line?

Reply:
> Is this message right when it is called from 958 line?

What's bothering you? It is assumed that the tablet is local and if there are problems connecting to it, it has probably gone away in the process. The aggregator will request information about it and try again.

Reply:
May be there could be more error reasons, not only "is not local".

Reply:
What's the point? What will the aggregator do with them?

Reply:
The point is only the message text)